### PR TITLE
Fix refreshing v5 cluster details

### PR DESCRIPTION
**Pull request by ograu**
Fixes https://github.com/giantswarm/giantswarm/issues/7560

Added animation for when adding node pools to the view

### CODE REVIEW

**Comment by ograu** [comment]
@oponder @axbarsan I would like to merge this just in case @marians have to use any of the other test . installations tomorrow. Do you want to make a look at at it?

File: src/components/cluster/detail/cluster_detail_view.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ class ClusterDetailView extends React.Component {
   registerRefreshInterval = () => {
     this.loadDataInterval = this.props.setInterval(
       this.loadDetails,
-      30 * 1000 // 30 seconds
+      3 * 1000 // 30 seconds
```
Comment (oponder) [inline src/components/cluster/detail/cluster_detail_view.js]: Is this intended to stay at 3? I think it was probably for testing right

File: src/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
...
+   azureCapabilitiesJSON: '{"Standard_A2_v2":{"additionalProperties":{},"description":"This is some description","maxDataDiskCount":4,"memoryInMb":4294.967296,"name":"Standard_A2_v2","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":21474.83648},"Standard_A4_v2":{"additionalProperties":{},"description":"Here is a longer description that might be too long for the field","maxDataDiskCount":8,"memoryInMb":8589.934592,"name":"Standard_A4_v2","numberOfCores":4,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":42949.67296},"Standard_A8_v2":{"additionalProperties":{},"description":"Another VM size description text","maxDataDiskCount":16,"memoryInMb":17179.869184,"name":"Standard_A8_v2","numberOfCores":8,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":85899.34592}, "Standard_D2s_v3":{"additionalProperties":{},"description":"Dsv3-series, general purpose, 160-190 ACU, premium storage supported","maxDataDiskCount":4,"memoryInMb":8589.935,"name":"Standard_D2s_v3","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":17179.87}}',
```
Comment (oponder) [inline src/index.html]: unfortunately I can't tell right away what changed here. But is it necessary for the topic of this PR?

Comment (oponder) [reply src/index.html]: looks like an indentation change?

Comment (ograu) [reply src/index.html]: It is changing now, I added instances to `awsCapabilitiesJSON` for testing and somehow I messed things a bit when removing them.

File: src/components/cluster/detail/cluster_detail_view.js (hunk 1 above)
Comment (ograu) [reply src/components/cluster/detail/cluster_detail_view.js]: Yeah, just for testing... thanks! 🤦‍♂

**Comment by ograu** [comment]
Not a lot of changes, actually, just the animation and a few things here and there. I think that this has more to do with the way git tracks changes

**Pull request merged**